protos/fabric.proto

```diff
@@ -107,10 +107,8 @@ message BlockchainInfo {
 // the block hash when verifying the blockchain.
 // localLedgerCommitTimestamp - The time at which the block was added
 // to the ledger on the local peer.
-// transactionResults - The results of transactions.
 message NonHashData {
     google.protobuf.Timestamp localLedgerCommitTimestamp = 1;
-    repeated TransactionResult transactionResults = 2;
```

Comment:
If the TransactionResult is removed, is there a reason to keep the TransactionResult message at line 67 or can that be removed also?